src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -96,4 +97,18 @@ export function getAuthenticatorDataForVerification (authenticator) {
 
 export function getIconClassNameForBeacon (authenticatorType) {
   return getAuthenticatorData({ authenticatorType }).iconClassName;
-}
\ No newline at end of file
+}
+
+export function removeRequirementsFromError (errorJSON, policy) {
+  const passwordRequirementsAsString = getPasswordComplexityDescription(policy);
+  if (errorJSON.errorCauses?.length > 0
+    && Array.isArray(errorJSON.errorCauses[0].errorSummary)
+    && errorJSON.errorCauses[0].errorSummary.length > 0) {
+
+    // Remove the [ and ] if they are present followed by requirements string if it is present.
+    errorJSON.errorCauses[0].errorSummary = errorJSON.errorCauses[0].errorSummary[0]
+      .replace(/[[\]]/g, '')
+      .replace(`${passwordRequirementsAsString}`, '').trim();
```

Comment:
Can we replace `passwordRequirementsAsString` first and them remove `[]`? Makes the logic more tighter.